src/WebJobs.Script/ScriptConstants.cs

```diff
@@ -214,6 +214,7 @@ public static class ScriptConstants
         public static readonly long DefaultMaxRequestBodySize = 104857600;
 
         public static readonly ImmutableArray<string> SystemLogCategoryPrefixes = ImmutableArray.Create("Microsoft.Azure.WebJobs.", "Function.", "Worker.", "Host.");
+        public static readonly ImmutableArray<string> SystemLogCategoryPrefixesForV3 = ImmutableArray.Create("Host.Startup");
```

Comment:
My only real concern here is that we'll lose backend telemetry that tells us real usage of our older versions. @mattchenderson I know I mentioned this off-hand, but do you know if we capture telemetry based on Metrics or if its based on FunctionStarted/Completed traces?

I'm nervous we'll see some drop and say that we've got almost no v3 usage, when in reality...

Reply:
Discussed offline